applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_windkessel_outlet_process.py

```diff
@@ -0,0 +1,138 @@
+import math
+import KratosMultiphysics
+from KratosMultiphysics.assign_scalar_variable_process import AssignScalarVariableProcess
+
+import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
+
+# Import applications
+import KratosMultiphysics.FluidDynamicsBiomedicalApplication as KratosBio
+
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ApplyWindkesselOutletProcess(Model, settings["Parameters"])
+
+
+class ApplyWindkesselOutletProcess(KratosMultiphysics.Process):
+    def __init__(self, Model, settings ):
+
+        KratosMultiphysics.Process.__init__(self)
+
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "mesh_id"            : 0,
+            "model_part_name"    : "",
+            "variable_name"      : "PRESSURE",
+            "constrained"        : true,
+            "value"              : 0.0,
+            "interval"           : [0.0,"End"],
+            "resistance1"        : 0.0,
+            "resistance2"        : 0.0,
```

Comment:
```suggestion
            "resistance_1"       : 0.0,
            "resistance_2"       : 0.0,
```
To follow our standard. Besides, if these are not customary ones in de literature (IDK), I'd try to use a more descriptive name for these two values to ease the use.